OCS/deploy-ocs.sh

```diff
@@ -2,6 +2,8 @@
 NAMESPACE="openshift-storage"
 LOCALNAMESPACE="local-storage"
 
+export KUBECONFIG=${KUBECONFIG:-../OpenShift/ocp/auth/kubeconfig}
```

Comment:
can you we go with something more fancy that .. ?

Reply:
sure, I'll follow up, just want to make this fixes the issue in automation

Reply:
Updated, could you please let me know how it looks?